vcpkg/commands/new.md

```diff
@@ -0,0 +1,145 @@
+---
+title: vcpkg new
+description: Command line reference for the vcpkg new command. Creates a new manifest in the current directory.
+ms.date: 07/05/2023
+---
+# vcpkg new
+
+## Synopsis
+
+```console
+vcpkg new --application
+```
+
+```console
+vcpkg new --name hello --version 1.0
+```
+
+## Description
+
+Creates a manifest file (`vcpkg.json`) and default configuration file (`vcpkg-configuration.json`)
+ready to use in your project.
+
+Use the `--application` flag if your project is an end-user application.
+
+If not using `--application`, `--name` and `--version` are required. `--version-date`,
+`--version-relaxed`, or `--version-string` can force that a particular version format is selected.
+For more information about the versioning formats, see
+[Versioning reference](../users/versioning.md).

```

Comment:
```suggestion
The `--name` and `--version` options are required, unless the `--application` flag is used. 

The `--version-date`, `--version-relaxed`, or `--version-string` flags force a particular version format.
For more information about the versioning formats, see the [versioning documentation](../users/versioning.md).
```

Reply:
That documents 'what the application flag does' but not 'why' so I'm not a fan of this framing. It sounds like it's a 'just shut the tool up switch' phrased this way.

In particular, I don't want to encourage things that are not applications skipping name and verison.